platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PassThroughWiring.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.wiring.components;
+
+import com.swirlds.common.wiring.model.WiringModel;
+import com.swirlds.common.wiring.schedulers.TaskScheduler;
+import com.swirlds.common.wiring.schedulers.builders.TaskSchedulerType;
+import com.swirlds.common.wiring.wires.input.BindableInputWire;
+import com.swirlds.common.wiring.wires.input.InputWire;
+import com.swirlds.common.wiring.wires.output.OutputWire;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.Objects;
+import java.util.function.Function;
+
+/**
+ * PassThrough Component Wiring, useful for wiring no-op components with the given scheduler type.
+ *
+ * @param <DATA_TYPE> the type of input and output for this component wiring
+ */
+public class PassThroughWiring<DATA_TYPE> {
+
+    private final BindableInputWire<DATA_TYPE, DATA_TYPE> input;
+    private final OutputWire<DATA_TYPE> output;
+    private final TaskScheduler<DATA_TYPE> scheduler;
+
+    /**
+     * PassThrough Constructor for creating a no-op component with the given scheduler type.
+     *
+     * @param model         the wiring model containing this input wire
+     * @param componentName the name of the component
+     * @param inputLabel    the label for the input wire
+     * @param schedulerType the type of scheduler to use
+     */
+    public PassThroughWiring(
```

Comment:
Pointing out that this constructor is unused, though it's fine if you'd like to keep it

Since the stuff apart from creating the scheduler matches the other constructor, it probably would make sense to call the other constructor from this one.

Reply:
One constructor calls the other, and usage of the unused constructor has been added to unit tests.